code-challenges.js

```diff
@@ -31,24 +38,66 @@ const temp2 = 350
 const temp3 = 212
 // Expected output: "212 is at boiling point"
 
+const temperature =(temp) => {
+    if (temp > 212) {
+        return `${temp} is above boiling point`
+    } else if (temp === 212) {
+        return `${temp} is at boiling point`
+    } else {
+        return `${temp} is below boiling point`
+    }
+}
+console.log(temperature(temp1));
+console.log(temperature(temp2));
+console.log(temperature(temp3));
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Describe your process:
+//Create a variable to combine the two arrays into a new array
+//Use a built-in method called .length to determine the length of the new array
+//Print the length of the new array
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+// let newPadresArray = padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns);
+// console.log(newPadresArray.length);
+const padres = (series1, series2) => {
+    let arrayLength = series1.concat(series2)
+    return arrayLength.length
+}
+console.log(padres(padres1984WorldSeriesRuns, padres1998WorldSeriesRuns));
+
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
 // Describe your process:
+//Create a new variable to use the mutator .split() in order to turn the string into an array since strings are immutable in JavaScript
+//Create a new variable to reverse the strings inside the new array
+//Print the variable you used to reverse the array and use the mutator .join() to change the array into a new string
 
 const currentCohort = "Golf 2022"
 // Expected output: "2202 floG"
 
+// let currentCohortArray = currentCohort.split("")
+// let reverseCurrentCohort = currentCohortArray.reverse();
+// console.log(reverseCurrentCohort.join(""))
+
+const cohort = (cohortName) => {
+    let currentCohortArray = cohortName.split("")
+    let reverseCurrentCohort = currentCohortArray.reverse()
+    return reverseCurrentCohort.join("")
+}
+console.log(cohort(currentCohort))
+
 // --------------------4) Create the code that will return the last index of the given value from the array using the test variables provided below.
 
 // Describe your process:
+//Input: given value
+//Process: Use the given value variable to reference the myNumbers array and iterate through until the given value matches what is found in the array
```

Comment:
I see iterate here in your pseudocode, but you end up using lastIndexOf.  It is ok to change the plan when you start coding, just be sure to add to your pseudocode the change in direction.